.github/workflows/build_wheels.yml

```diff
@@ -47,9 +47,11 @@ jobs:
         CIBW_BEFORE_BUILD: "git submodule foreach git clean -fdx"
         # The embedded 'tx' C executable is independent of Python ABI so we only need to build it once.
         # Here we choose the minimum supported version, i.e. Python 3.6
-        CIBW_BUILD: "cp36-*"
```

Comment:
(maybe in another PR) we should drop 3.6 at some point as it's EOL. `python_requires` in setup.py should also be updated so pip doesn't download the incompatible version.